sql/core/src/main/scala/org/apache/spark/sql/execution/metric/SQLMetrics.scala

```diff
@@ -29,9 +29,18 @@ class SQLMetric(val metricType: String, initValue: Long = 0L) extends Accumulato
   // We may use -1 as initial value of the accumulator, if the accumulator is valid, we will
   // update it at the end of task and the value will be at least 0. Then we can filter out the -1
   // values before calculate max, min, etc.
-  private[this] var _value = initValue
+  private var _value = initValue
```

Comment:
`initValue` will be assigned to `_value` when creating accumulator, and `zeroValue` will be assigned to `_value` when calling `reset`, how about it?

Most of the time `initValue` is equal to `zeroValue`,  it's only used in `copy`.